website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -124,7 +124,8 @@ curl \
         "fair-run-queuing-enabled": true,
         "saml-enabled": false,
         "owners-team-saml-role-id": null,
-        "two-factor-conformant": false
+        "two-factor-conformant": false,
```

Comment:
Can we add a note under the Sample Response heading that these two are only available for TFC. Maybe something like:

**Note:** The `two-factor-conformant` and `enforce-assessments` properties are only returned from Terraform Cloud organizations.